### PR TITLE
Add javigon

### PR DESCRIPTION
**Pull request by jesusvallejo**

### CODE REVIEW

File: csgo_bot.py [hunk 1]
```diff
@@ -1,18 +1,16 @@
-# import pdb
+﻿# import pdb
```
Comment (mrpondere) [inline csgo_bot.py]: Not needed

File: csgo_bot.py [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
 from db_helper import DBHelper
-
+from telebot import types
 ############################################
 #                 DATOS                    #
 ############################################
 reload(sys)
 
-TOKEN = os.environ.get('csgo_bot_token')
+TOKEN = '526852229:AAFe-wyzOBxXYD3PxEJCvt_vwPlDurLvcxs' #os.environ.get('csgo_bot_token')
```
Comment (mrpondere) [inline csgo_bot.py]: Idem

File: csgo_bot.py [hunk 3]
```diff
@@ -21,7 +19,7 @@
 
 # test -1001280311618
 # CSGO -1001107551770
-GROUP_ID = -1001107551770
+GROUP_ID = -319789223
```
Comment (mrpondere) [inline csgo_bot.py]: It's better if we keep the csgo group.

File: csgo_bot.py [hunk 4]
```diff
@@ -102,6 +100,51 @@ def list_mix(message):
     items = db.get_items()
     bot.send_message(chat_id, items, parse_mode='Markdown')
 
+    
+	
+
```
Comment (mrpondere) [inline csgo_bot.py]: Too many lines between functions

File: csgo_bot.py [hunk 5]
```diff
@@ -102,6 +100,51 @@ def list_mix(message):
     items = db.get_items()
     bot.send_message(chat_id, items, parse_mode='Markdown')
 
+    
+	
+	
+@custom_group_only
+def get_javigon(message):
+	try:
```
Comment (mrpondere) [inline csgo_bot.py]: I think we should follow pep8 identation style [1]

[1] https://www.python.org/dev/peps/pep-0008/#indentation

File: csgo_bot.py [hunk 6]
```diff
@@ -102,6 +100,51 @@ def list_mix(message):
...
+		
+	except:
+		print("Oops!Try again...")
+	
+	
+@custom_group_only
+def get_sinDuda(message):
+	try:
+		chat_id = message.chat.id
+		#print("eliminando teclado")
```
Comment (mrpondere) [inline csgo_bot.py]: Not needed

File: csgo_bot.py [hunk 7]
```diff
@@ -102,6 +100,51 @@ def list_mix(message):
...
+	except(Exception, ArithmeticError) as e:
+		print("Oops! Archivo javigon_sinDuda_audio.ogg not found.")
+		bot.send_message(chat_id, "Audio no encontrado, contacta con Admin", reply_markup=markup)
+
+
+@custom_group_only
+def get_pollon(message):
+	try:
+		chat_id = message.chat.id
+		#print("eliminando teclado")
```
Comment (mrpondere) [inline csgo_bot.py]: Idem

File: csgo_bot.py [hunk 8]
```diff
@@ -102,6 +100,51 @@ def list_mix(message):
...
+	
+@custom_group_only
+def get_sinDuda(message):
+	try:
+		chat_id = message.chat.id
+		#print("eliminando teclado")	
+		markup = types.ReplyKeyboardRemove(selective=False)
+		bot.send_message(chat_id, "ok reproduciendo sinDuda", reply_markup=markup)
+		bot.send_audio(chat_id=chat_id, audio=open('javigon_sinDuda_audio.ogg', 'rb'))
+		bot.send_message(chat_id, "ok reproduciendo sinDuda", reply_markup=markup)
```
Comment (mrpondere) [inline csgo_bot.py]: Why are you sending the same message twice?

File: csgo_bot.py (hunk 1 above)
Comment (jesusvallejo) [reply csgo_bot.py]: ok ill remove it

File: csgo_bot.py (hunk 3 above)
Comment (jesusvallejo) [reply csgo_bot.py]: i cant debug it in csgo group , ill post it with the csgo group in the oncoming commits

File: csgo_bot.py (hunk 4 above)
Comment (jesusvallejo) [reply csgo_bot.py]: ill review the indentation

File: csgo_bot.py (hunk 6 above)
Comment (jesusvallejo) [reply csgo_bot.py]: new commit, please review it

File: csgo_bot.py (hunk 7 above)
Comment (jesusvallejo) [reply csgo_bot.py]: new commit, please review it

File: csgo_bot.py (hunk 8 above)
Comment (jesusvallejo) [reply csgo_bot.py]: solved

File: csgo_bot.py (hunk 2 above)
Comment (jesusvallejo) [reply csgo_bot.py]: solved

File: csgo_bot.py [hunk 9]
```diff
@@ -1,12 +1,9 @@
...
-import sys
+﻿import sys
 import os
 from importlib import reload
-
 import telebot
-
 from db_helper import DBHelper
-
+from telebot import types
```
Comment (mrpondere) [inline csgo_bot.py]: According to [1]  we should leave the structure like:

import sys
import os
from importlib import reload

import telebot
from telebot import types (why not using directly telebot.types in your calls?)

from db_helper import DBHelper


[1] https://stackoverflow.com/questions/22722976/import-order-coding-standard

File: csgo_bot.py [hunk 10]
```diff
@@ -15,12 +12,13 @@
 TOKEN = os.environ.get('csgo_bot_token')
 
 bot = telebot.TeleBot(TOKEN)
-
+evitar_flood_javigon=True #It avoids flooding with commands , True to activate(default False)
```
Comment (mrpondere) [inline csgo_bot.py]: Why not make  a more general variable so we can reuse it?

```python
bot_settings = { 
    'flood_javigon': True,
}
```

File: csgo_bot.py [hunk 11]
```diff
@@ -15,12 +12,13 @@
...
 db = DBHelper()
 db.setup()
 
 # test -1001280311618
 # CSGO -1001107551770
+# test jesus -319789223
```
Comment (mrpondere) [inline csgo_bot.py]: not usefull

File: csgo_bot.py [hunk 12]
```diff
@@ -102,6 +100,92 @@ def list_mix(message):
     items = db.get_items()
     bot.send_message(chat_id, items, parse_mode='Markdown')
 
+    
+@custom_group_only
```
Comment (mrpondere) [inline csgo_bot.py]: With pep8 you will end with something like:

```python

@custom_group_only
def get_javigon(message):
    try:
        chat_id = message.chat.id
	markup = types.ReplyKeyboardMarkup(row_width=2)
	itembtn1 = types.KeyboardButton('/sinDuda')
	itembtn2 = types.KeyboardButton('/pollon')
	markup.add(itembtn1, itembtn2)
	bot.send_message(chat_id, "Elige tu audio:", reply_markup=markup)
    except Exception:
        print("Oops! Try again...")
```

File: csgo_bot.py [hunk 13]
```diff
@@ -102,6 +100,92 @@ def list_mix(message):
...
+def get_sinDuda(message):
+	chat_id = message.chat.id
+	if evitar_flood_javigon:#evita el flood
+		try:
+			bot.delete_message(message.chat.id,
+                    		           message.message_id-1)#elimina los mensajes para evitar flood
+			bot.delete_message(message.chat.id, 
+			                   message.message_id)
+
+		except(Exception, ArithmeticError) as e:
```
Comment (mrpondere) [inline csgo_bot.py]: Why catching ArithmeticError?

**Pull request merged**